CHANGELOG.md

```diff
@@ -633,7 +633,7 @@
     * flant-statusmap-panel
     * btplc-status-dot-panel
   * [CHANGE] [cert-manager](https://cert-manager.io/docs/usage/certificate/) resources
-  now use 'v1' to align with their use in SAS Viya 4.x
+  now use 'v1' to align with their use in SAS Viya platform 4.
```

Comment:
See general comments.  But, in any case, I suspect "SAS Viya platform 4" is never correct.